spec/grammar-spec.coffee

```diff
@@ -508,3 +508,11 @@ $asd\\n
       expect(lines[2][1]).toEqual value: "key", scopes: ["source.perl", "string.quoted.single.perl"]
       expect(lines[2][2]).toEqual value: "'", scopes: ["source.perl", "string.quoted.single.perl", "punctuation.definition.string.end.perl"]
       expect(lines[2][3]).toEqual value: " ", scopes: ["source.perl"]
+
+  describe "when package to tokenizes", ->
+    it "does not highlight semicolon in package name", ->
+      {tokens} = grammar.tokenizeLine("package Test::ASD;")
```

Comment:
Would you mind adding a trailing comment to this line and checking the tokens have the right comment scopes? Just to make sure nothing regressed